tests/python_api/tf_classification_reader.py

```diff
@@ -28,17 +28,13 @@
 from parse_config import parse_args
 
 
-def draw_patches(img, idx, device_type, args=None):
+def draw_patches(img, idx, device, args=None):
     import cv2
     args = parse_args()
-    if device_type == "gpu":
-        try:
-            import cupy as cp
-            img = cp.asnumpy(img)
-        except ImportError:
-            pass
     if not args.NHWC:
         img = img.transpose([0, 1, 2])
+    if device == "gpu":
```

Comment:
@LakshmiKumar23 Please move the device==gpu check to L34. Running the TF classification reader example with NCHW layout throws this error
```
AttributeError: EagerTensor object has no attribute 'transpose'. 
        If you are looking for numpy-related methods, please run the following:
        tf.experimental.numpy.experimental_enable_numpy_behavior()
```